test/sanity/run-test.sh

```diff
@@ -32,6 +32,12 @@ function install_csi_sanity_bin {
   popd
 }
 
+function provision_nfs_server {
```

Comment:
move this step to github action only.

Reply:
if we move this to GH actions workflow, we may not be able to reuse it for prow

Reply:
can we leave this here while we're experimenting with DinD for Prow?

Reply:
ok, keep it.